web/src/beta/components/Resizable/index.tsx

```diff
@@ -33,24 +41,50 @@ const Resizable: React.FC<Props> = ({
   const BottomGutter = showBottomGutter ? <HorizontalGutter {...gutterProps} /> : null;
   const LeftGutter = showLeftGutter ? <VerticalGutter {...gutterProps} /> : null;
 
+  useEffect(() => {
+    if (size <= initialSize / 2) {
+      setMinimized(true);
+    } else {
+      setMinimized(false);
+    }
+  }, [direction, initialSize, size]);
+
+  const handleShowOriginalPanel = useCallback(() => {
+    setMinimized(false);
+    onInitializeSize();
+  }, [onInitializeSize]);
```

Comment:
move the minimize useState in to the hooks and do this all in the original callback 🙏🏻

Reply:
done

Reply:
You only did the first part to this comment. You moved minimized in to the hooks, but you didn't handle setting minimized to false there too. If you do that, you can delete this callback

Reply:
done